mfa/yubikey_darwin_amd64.go

```diff
@@ -0,0 +1,102 @@
+package mfa
```

Comment:
Needs a build tag?

Reply:
👍 Ah yep cool. So we can drop the amd64? `yubikey_darwin.go`

Reply:
It's `_$GOOS_$GOARCH.go`, I don't think are there any non amd64 darwin machines around but won't hurt to keep it as it is.